owslib/ogcapi/edr.py

```diff
@@ -0,0 +1,98 @@
+# =============================================================================
+# Copyright (c) 2023 Tom Kralidis
+#
+# Author: Tom Kralidis <tomkralidis@gmail.com>
+#
+# Contact email: tomkralidis@gmail.com
+# =============================================================================
+
+from io import BytesIO
+import logging
+from typing import BinaryIO
+
+from owslib.ogcapi.features import Features
+from owslib.util import Authentication
+
+LOGGER = logging.getLogger(__name__)
+
+
+class EnvironmentalDataRetrieval(Features):
+    """Abstraction for OGC API - Envirionmental Data Retrieval"""
+
+    def __init__(self, url: str, json_: str = None, timeout: int = 30,
+                 headers: dict = None, auth: Authentication = None):
+        __doc__ = Features.__doc__  # noqa
+        super().__init__(url, json_, timeout, headers, auth)
+
+    def data(self) -> list:
+        """
+        implements /collections filtered on EDR data resources
+
+        @returns: `list` of filtered collections object
+        """
+
+        datas = []
+        collections_ = super().collections()
+
+        for c_ in collections_['collections']:
+            for l_ in c_['links']:
+                if 'data' in l_['rel']:
+                    datas.append(c_['id'])
+                    break
+
+        return datas
+
+    def query_data(self, collection_id: str,
+                   query_type: str, **kwargs: dict) -> BinaryIO:
+        """
+        implements /collection/{collectionId}/coverage/
```

Comment:
From someone who's not familiar with the API, I suggest the docstring should start with what the function does in simple terms.